src/app.js

```diff
@@ -1,9 +1,20 @@
 const express = require('express');
+const artistControllers = require('./controllers/artists.js');
+const albumControllers = require('./controllers/albums.js');
+const songControllers = require('./controllers/songs');
 
 const app = express();
+app.use(express.json());
 
-app.get('*', (req, res) => {
-  res.status(200).json({ message: 'Hello World!' });
-});
+app.post('/artists', artistControllers.create);
+app.get('/artists', artistControllers.list);
+app.get('/artists/:artistId', artistControllers.find);
+app.patch('/artists/:artistId', artistControllers.patch);
+app.delete('/artists/:artistId', artistControllers.delete);
+app.post('/artists/:artistId/albums', albumControllers.create);
+app.get('/artists/:artistId/albums', albumControllers.list);
+app.patch('/artists/:artistId/albums/:albumId', albumControllers.patch);
+app.delete('/artists/:artistId/albums/:albumId', albumControllers.delete);
+app.post('/albums/:albumId/song', songControllers.create);
 
```

Comment:
All good here. The next step would be to take these out and put them into `router` components. Let me know if you need an example.